data/presets/highway/motorway_junction.json

```diff
@@ -13,5 +13,5 @@
     "terms": [
         "exit"
     ],
-    "name": "Motorway Junction / Exit"
+    "name": "Motorway Junction/Exit"
```

Comment:
I think this should be simplified to just “Motorway Exit” in the American English localization. Although longtime OSM contributors may be used to seeing the phrase “motorway junction”, this is a very British term that could lead to misunderstanding among newer mappers.

In the U.S., a junction would be any place where a major road connects to another major road, even indirectly via ramps, but not the specific location where `highway=motorway_junction` needs to be placed. We call those “(highway/freeway) exits”, as opposed to “entrances”. I’ve often encountered `highway=motorway_junction` incorrectly placed at a freeway entrance, potentially throwing off routers like OSRM that count exits along a step. This preset name might be a factor in that misuse.

(I’d even go a step further and advocate for replacing “motorway” with a non-British term, but that’s a separate discussion.)

Reply:
#893